src/utils/logger.ts

```diff
@@ -40,6 +51,10 @@ export const Logger = new (class Logger implements vscode.Disposable {
     this.log(LogLevel.DEBUG, message, optionalParams);
   }
 
+  process(message: string): void {
+    this.outputChannel.appendLine(`${message}\n`);
```

Comment:
I think it's better to have a prefix or something. this might cause a lot of noise and we should be able to tell if that a binary log or the extension log